src/fable-fsharp/Replacements/Replacements.fs

```diff
@@ -1092,6 +1092,9 @@ module private CoreLibPass =
             fsharp + "Collections.Seq" => ("Seq", Static)
             fsharp + "Collections.Set" => ("Set", Static)
             fsharp + "Core.Choice" => ("Choice", Both)
+            fsharp + "Control.Event" => ("Event", Both)
```

Comment:
I think `Both` is needed for `Event.Publish` and `Event.Trigger` functions which are class methods not static functions on module... ( I think I  `Static` was my 1st choice and it failed on those functions)